expr/sort.go

```diff
@@ -367,6 +367,14 @@ func LookupCompare(typ zed.Type) comparefn {
 			return va.Compare(vb)
 		}
 
+	case zed.IDType:
+		zctx := zed.NewContext()
+		return func(a, b zcode.Bytes) int {
+			va, _ := zctx.DecodeTypeValue(a)
+			vb, _ := zctx.DecodeTypeValue(b)
+			return zed.CompareTypes(va, vb)
+		}
```

Comment:
I think we ultimately want `func CompareTypeValue(a, b zcode.Bytes) int` because `DecodeTypeValue` isn't cheap, but this is fine for now!

Reply:
Agreed!

Reply:
I'll put a comment here.